<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Warthog Network</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 156), stop:0.00 rgba(240, 150, 80, 210), stop:0.9999 rgba(0, 0, 0, 240), stop:1 rgba(0, 0, 0, 0));
</string>
   </property>
   <widget class="QPushButton" name="select_button">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 10px;
font-size: 24px;
background-color: rgba(255,255,255,80);
border: 1px solid rgba(255,255,255,40);
font-weight: bold;
color: #DCDCDC;
}
QPushButton:hover{
background-color: rgba(255,255,255,60);
}
</string>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_text">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>21</y>
      <width>501</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;
font-size: 24px;
background-color: rgba(255,255,255,70);
border: 1px solid rgba(255,255,255,40);
color:black;</string>
    </property>
   </widget>
   <widget class="QListWidget" name="items">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>91</y>
      <width>661</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: rgba(255,255,255,90);
border: 1px solid rgba(255,255,255,40);
color:black;
</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
